integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/nifi/v2/init/InitializeFeed.java

```diff
@@ -140,10 +155,13 @@
 
     private Map<String, AtomicInteger> retryCounts = Collections.synchronizedMap(new HashMap<>());
 
+    private CleanupEventService cleanupEventService;
+
     @OnScheduled
     public void scheduled(ProcessContext context) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)